src/main/kotlin/com/terraformation/backend/accelerator/api/ProjectModulesController.kt

```diff
@@ -84,11 +87,12 @@ data class ProjectModule(
       overview = model.overview,
       preparationMaterials = model.preparationMaterials,
       events =
-          model.eventDescriptions.mapValues {
+          model.eventDescriptions.map {
```

Comment:
This might be clearer with explicit variable names, e.g., `.map { (eventType, description) ->` rather than using `it.key` and `it.value`.